runtime/testy/runtime.py

```diff
@@ -103,17 +117,111 @@ def startStateManager(badThingsQueue, stateQueue, runtimePipe):
   except Exception:
     badThingsQueue.put(BadThing(sys.exc_info(), None))
 
-def processFactory(badThingsQueue, stateQueue):
-  def spawnProcessHelper(processName, helper):
+def processFactory(badThingsQueue, stateQueue, stdoutRedirect = None):
+  def spawnProcessHelper(processName, helper, *args):
     pipeToChild, pipeFromChild = multiprocessing.Pipe()
     if processName != PROCESS_NAMES.STATE_MANAGER:
       stateQueue.put([SM_COMMANDS.ADD, [processName, pipeToChild]], block=True)
       pipeFromChild.recv()
-    newProcess = multiprocessing.Process(target=helper, name=processName.value, args=(badThingsQueue, stateQueue, pipeFromChild))
+    newProcess = multiprocessing.Process(target=helper, name=processName.value, args=[badThingsQueue, stateQueue, pipeFromChild] + list(args))
     allProcesses[processName] = newProcess
     newProcess.daemon = True
     newProcess.start()
   return spawnProcessHelper
 
+def testTime():
```

Comment:
Name this something more descriptive please, makes it seem like you're testing the time module